sdks/java/core/src/test/java/org/apache/beam/sdk/runners/inprocess/EncodabilityEnforcementFactoryTest.java

```diff
@@ -56,7 +56,7 @@
   public void encodeFailsThrows() {
     TestPipeline p = TestPipeline.create();
     PCollection<Record> unencodable =
-        p.apply(Create.of(new Record()).withCoder(new RecordNoEncodeCoder()));
+        p.apply(Create.<Record>of().withCoder(new RecordNoEncodeCoder()));
```

Comment:
Can you do this more faithfully (or perhaps additionally) by adding a test with `Create.of(t)` -> `ParDo.of(emit uncodable)`?